lib/python/rose/popen.py

```diff
@@ -296,7 +296,7 @@ def which(name):
             if kwargs.get("shell"):
                 proc = Popen(args[0], **kwargs)
             else:
-                command = ' '.join(args)
+                command = ' '.join(map(shlex.quote, args))
```

Comment:
@matthewrmshin could you make sure you're OK with this and re-approve.

Reply:
OK with this.